### ISSUE TITLE
[a11y] [WCAG 1.4.1]: Checkbox controls are not visible after applying high contrast

### ISSUE THREAD

**Issue by dkatzz** [civiform/civiform#9913]
Observed Results:
--
After applying high contrast while navigating over the checkbox elements, checkbox controls are not visible while navigating using TAB key.

Screencast: Chromebook[here](https://drive.google.com/file/d/19vcrKPNcER096kdjImP3nAHDJJ1b44WZ/view?usp=drive_link ) \
Screencast: Windows[here]( https://screencast.googleplex.com/cast/NjA3Mjg1MDU0Nzk5ODcyMHwyZTA5ZjkzYS1jYg )

User Impact:
----------
Users with color deficiency will face difficulty if checkbox controls are not visible for selecting while navigating in dark mode

Expected Results:
--
While navigating checkbox controls under "Available programs and services" region in high contrast mode, checkbox controls should clearly visible

Steps to Reproduce:
-------
1. Launch the URL[here]( https://ns.civiform.dev/programs )
2. Open developers tools - more options - select rendering option
3. Select "prefers - colors-scheme-dark" under "Emulate CSS media feature prefers-colour-scheme" in rendering view
4. Select "forced-colors:active" under "Emulate CSS media feature forced-colors" in rendering view
5. Now navigate over the checkbox elements and verify checkbox controls in dark mode

WCAG Details:
--
[WCAG 1.4.1][Use-of-color]( https://www.w3.org/WAI/WCAG22/quickref/?versions=2.0#use-of-color )

Additional Information
--
N/A

Environment Details:
--
Machine: chromebook 13.3/Windows \
Browser: Version 132.0.6834.208 (Official Build) (64-bit) \
Screen Reader : NA \
Bug Filling Time: 15 mins.
